<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="1" rowspan="2">
     <widget class="QPushButton" name="run_test_btn">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="export_result_btn">
      <property name="text">
       <string>Export Result</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="show_all_btn">
        <property name="text">
         <string>Show All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_selected_btn">
        <property name="text">
         <string>Show Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_unselected_btn">
        <property name="text">
         <string>Show Unselected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="clear_result_btn">
      <property name="text">
       <string>Clear Result</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="passed_label">
      <property name="text">
       <string>Passed:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="import_tests_btn">
      <property name="text">
       <string>Import Tests</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="selected_count_label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>0 test(s) Selected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="toggle_all_btn">
      <property name="text">
       <string>Unselect/Select All</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="failed_label">
      <property name="text">
       <string>Failed:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="8">
     <widget class="QTableWidget" name="test_table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>Test</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Result</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QListWidget" name="result_log_list">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
